assets/js/models/service_record.ts

```diff
@@ -0,0 +1,6 @@
+export interface ServiceRecord {
+  route: string;
```

Comment:
I'd hesitate to call this `route` because it doesn't map directly onto the GTFS concept of a route. It _does_ map pretty directly onto our GTFS concept of a `line`, though (the Green Line routes together are `line-Green`, 741+742+743+746 are `line-SLWaterfront`). It would be nice if we could actually fetch the data based on Lines rather than Routes, but that would add a bit more complexity since it's not possible to directly filter on Lines, and the frequency of changes to the line/route mapping for these specific routes should be very low.

Reply:
Yeah, given the extremely low data size and update frequency I'm inclined to just leave these hardcoded. I'll rename the field to `line` though.